### PR TITLE
Add JDBC events for ManageIQ into command gateway

### PR DESCRIPTION
**Pull request by tumido**
Add events into the command gateway so they are  visible to ManageIQ

Related to: https://github.com/ManageIQ/manageiq-providers-hawkular/pull/71

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

**Comment by theute** [comment]
@jshaughn I guess this is for you

File: hawkular-rest/hawkular-listener/src/main/java/org/hawkular/listener/bus/CommandEventListener.java [hunk 1]
```diff
@@ -71,6 +71,16 @@ protected void onBasicMessage(BasicMessage msg) {
...
+            case "AddJdbcDriverResponse": {
+                miqEventType = "hawkular_jdbc";
+                miqResourceType = "MiddlewareServer";
+                break;
+            }
+            case "RemoveJdbcDriverResponse": {
+                miqEventType = "hawkular_jdbc_remove";
+                miqResourceType = "MiddlewareServer";
+                break;
+            }
```
Comment (jshaughn) [inline hawkular-rest/hawkular-listener/src/main/java/org/hawkular/listener/bus/CommandEventListener.java]: The additions look good.  Nitpick:  if you don't mind, maintain the alphabetical ordering in the cases.

Comment (tumido) [reply hawkular-rest/hawkular-listener/src/main/java/org/hawkular/listener/bus/CommandEventListener.java]: Sure. I've noticed that it's in alphabetical order.. and then decided to ignore it. I don't know why.

**Comment by jshaughn** [comment]
Thanks @tumido.

**Pull request merged**